src/terraform/structure/terraform_parser.go

```diff
@@ -45,6 +45,7 @@ type TerrraformParser struct {
 	moduleImporter         *command.GetCommand
 	moduleInstallDir       string
 	downloadedPaths        []string
+	lock                   sync.Mutex
```

Comment:
@matansha Can we give this lock a more meaningful name please?
For example - `tfClientLock`

Reply:
@nimrodkor fixed, thanks!